packages/server/src/api/courseRoutes.ts

```diff
@@ -0,0 +1,39 @@
+import { CourseModel } from "../entity/CourseModel";
+import { GetCourseResponse } from "@template/common";
+import { ServerRoute } from "@hapi/hapi";
+import Joi from "@hapi/joi";
+import { pick } from "lodash";
+
+export const courseRoutes: ServerRoute[] = [
+  {
+    method: "GET",
+    path: "/api/v1/courses/{course_id}",
+    handler: async (request, h): Promise<GetCourseResponse> => {
+      const course = await CourseModel.findOne(request.params.course_id, {
+        relations: ["officeHours"],
+      });
+      return {
+        name: course.name,
```

Comment:
Would we also want to include the `course.id` here?

Reply:
we figured not, since you'll already know the course id if you're querying the backend for it.